torch/csrc/jit/codegen/cuda/test/test_gpu2.cpp

```diff
@@ -4191,8 +4191,8 @@ TEST_F(NVFuserTest, FusionVectorizeMisalignedPointwiseMergeSymbolicPass_CUDA) {
   constexpr int kVecSize = 2;
   constexpr int kNumElems = kTDX * kVecSize;
 
-  auto tv0 = makeSymbolicTensor(kNumDims);
-  auto tv1 = makeSymbolicTensor(kNumDims);
+  auto tv0 = makeContigTensor(kNumDims);
```

Comment:
Haven't looked at the changes of this PR yet, but why were these changes made? Does this PR have anything to do with MisalignedVectorize?

Reply:
Tried the tests in this file. It seems none needs to be changed except for `FusionVectorization3`.

Reply:
I don't remember, doesn't misaligned vectorization recheck the start boundary for every row independently? Maybe that was my poor assumption.

Reply:
Even if it did, if the inner most stride wasn't 1 then we couldn't use it right?

Reply:
There's some runtime checks for misaligned vectorization, and I think that the innermost stride is one of them. So, it should fail at the run time if a non-stride-1 tensor was given.